src/styles/variables.json

```diff
@@ -730,18 +730,36 @@
           }
         },
         "space": {
-          "gap": {
-            "default": "0",
-            "borderless": "0.25rem"
+          "panel": {
+            "gap": {
+              "default": "0.25rem",
+              "borderless": "0.25rem"
+            },
+            "default": {
+              "x": "3px",
+              "y": "3px"
+            },
+            "borderless": {
+              "x": "0",
+              "y": "0"
+            }
+          },
+          "default": {
```

Comment:
I'm a bit concerned that we have static fields (panel, gap) and variables (default, borderless) on the same level. Can we make a structure like this?
```
space: {
  panel: {
    gap: { default, borderless }
    padding: { default, borderless }
  }
  button: { default, borderless }
}
```